layout/MDErgo1-Default-0.kll

```diff
@@ -65,8 +68,8 @@ U"K" : U"K";
 U"L" : U"L";
 U"SEMICOLON" : U"SEMICOLON";
 U"QUOTE" : U"QUOTE";
-U"RALT" : U"MINUS";
-U"RCTRL" : U"EQUAL";
```

Comment:
putting `-` and `=` on the thumb keys wasn't a great idea since I use these really often. modifier keys and macros might generally be a better fit for the thumb switches.